webui/react/src/pages/Settings/UserManagement.tsx

```diff
@@ -126,12 +101,34 @@ const UserActionDropdown = ({ fetchUsers, user, groups, userManagementEnabled }:
           ]
       : [{ key: MenuKey.View, label: 'View User' }];
 
+  const handleDropdown = useCallback(
+    async (key: string) => {
+      switch (key) {
+        case MenuKey.Agent:
+          ConfigureAgentModal.open();
+          break;
+        case MenuKey.Edit:
+          EditUserModal.open();
+          break;
+        case MenuKey.Groups: {
+          const response = await getGroups({ userId: user.id });
+          setSelectedUserGroups(response.groups ?? []);
+          ManageGroupsModal.open();
+          break;
+        }
+        case MenuKey.State:
+          await onToggleActive();
+          break;
+        case MenuKey.View:
```

Comment:
nit: looks like you left out `ViewUserModal.open();`

Reply:
oops thanks for catching that